### ISSUE TITLE
[BUG] `prefer-commenting-analyzer-ignores` doesn't work if description contains `-`

### ISSUE THREAD

**Issue by ookami-kb** [dart-code-checker/dart-code-metrics#894]
* **Dart code metrics version:** 4.15.1
* **Dart sdk version:** 2.17.1

**What did you do? Please include the source code example causing the issue.**

```dart
// ignore: avoid-non-null-assertion, checked for non-null
description!,
```

**What did you expect to happen?**

No `prefer-commenting-analyzer-ignores` warning.

**What actually happened?**

The warning is shown. If I remove `-` from the description (e.g. `// ignore: avoid-non-null-assertion, checked for null`) no warning is shown.

**Are you willing to submit a pull request to fix this bug?**

Not sure.

**Comment by incendial** [comment]
Available in 4.16.0! 🚀
